targets/TARGET_NXP/TARGET_MCUXpresso_MCUS/TARGET_IMX/trng_api.c

```diff
@@ -0,0 +1,54 @@
+/*
+ *  Hardware entropy collector for the K66F, using Freescale's RNGA
+ *
+ *  Copyright (C) 2006-2015, ARM Limited, All Rights Reserved
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may
+ *  not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+/*
+ * Reference: "K66 Sub-Family Reference Manual, Rev. 2", chapter 38
```

Comment:
Could we please correct this reference to "i.MX RT1050 Processor ReferenceManual" Section 6.6?

Reply:
Thank you for spotting this, I have updated the file to correct this.